Tests/Data/Parabolic/T/3D_Beier_sandbox/beier_sandbox.prj

```diff
@@ -119,6 +110,16 @@
                 </phase>
             </phases>
             <properties>
+				<property>
+                    <name>porosity</name>
+                    <type>Constant</type>
+                    <value>0</value>
+                </property>
+				<property>
```

Comment:
tabs are not allowed. Use four spaces.

Reply:
Better use the xmlstarlet tool on the entire file like described above.

Reply:
Tabs are replaced. All project files have been reformatted using xmlstarlet.